packages/terra-notification-dialog/src/NotificationDialog.module.scss

```diff
@@ -8,6 +8,10 @@
     flex-direction: column;
     position: fixed;
 
+    //The query checks whether the browser is IE or not.
+    @media screen and (-ms-high-contrast: active), (-ms-high-contrast: none) {
```

Comment:
Do we need to check for max-width: 768px as well here ?

Reply:
As per MDN doc it looks like this media feature is not suggested one https://developer.mozilla.org/en-US/docs/Web/CSS/@media/-ms-high-contrast

Reply:
> Do we need to check for max-width: 768px as well here ?

I have added the max-width to be 768px for this query.